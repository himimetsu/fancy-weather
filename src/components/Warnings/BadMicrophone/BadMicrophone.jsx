import React from 'react';
import './BadMicrophone.css';

const BadMicrophone = ({ lang }) => {
  return (
    <div className='bad-geo'>
      {lang === 'ru'
        ? ('Пожалуйста, предоставьте сайту доступ к вашему микрофону!')
        : lang === 'en'
          ? ('Please, grant the site access to your microphone!')
          : ('Будь ласка, надайте сайту доступ до Вашого мікрофону!')
      }
    </div>
  );
};

export default BadMicrophone;
