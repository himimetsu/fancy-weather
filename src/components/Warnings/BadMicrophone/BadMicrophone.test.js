import React from 'react';
import ReactDOM from 'react-dom';
import BadMicrophone from './BadMicrophone';
import { render, cleanup } from '@testing-library/react';

afterEach(cleanup);

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<BadMicrophone />, div);
});

it('mathes snapshot', () => {
  const { asFragment } = render(<BadMicrophone />);
  expect(asFragment()).toMatchSnapshot();
})