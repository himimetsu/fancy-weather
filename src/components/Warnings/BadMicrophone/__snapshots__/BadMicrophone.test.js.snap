// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`mathes snapshot 1`] = `
<DocumentFragment>
  <div
    class="bad-geo"
  >
    <span
      class="ru"
    >
      Пожалуйста, предоставьте сайту доступ к вашему микрофону!
    </span>
    <span
      class="en"
    >
      Please, grant the site access to your microphone!
    </span>
    <span
      class="be"
    >
      Калі ласка, падайце сайту доступ да вашага мікрафону!
    </span>
  </div>
</DocumentFragment>
`;
